### ISSUE TITLE
I want to know how to download more than 2 files from FastAPI server.

### ISSUE THREAD

**Issue by jbc0708** [tiangolo/fastapi#5000]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
No Code..
Currently, it is sent to the client as a compressed file(ZipFile).
If it is an image, it will reply with a FileResponse
```

### Description

If a client requests multiple images with the keyword “computer”, 
is it possible to send multiple files at once without a compressed file?

### Operating System

macOS

### FastAPI Version

0.68

### Python Version

3.10

**Comment by JarroVGIT** [comment]
No, afaik you cannot return multiple files natively without client logic. For example, you could concatenate files together with an agreed upon Seperator and send it as one “file”, but you would have to unconcatenate it on the client side.

**Comment by jbc0708** [comment]
> 

Hi Jarro~ Thanks to answer for me.. So.. mean what you say is Zip file is best choice.. isn't it..?

**Comment by JarroVGIT** [comment]
No, the body of your response is either a json body or bytes. In your case, a zip file would be the best response. You could do some hacky solution though, by setting some custom headers containing information you wanted to relay via a json body? Anyway, this is not really a FastAPI issue i guess.

**Comment by jbc0708** [comment]
>
Thank you answer..

**Comment by jbc0708** [comment]
> You need client side logic, [StreamingResponse](url) maybe can help you, when I return documents like `.csv` I usually do
> 
> ```python
>   return StreamingResponse(
>       buffer, # BytesIO Object to return
>       media_type="text/csv",
>       headers={
>           "Content-Disposition": "attachment;filename=contacts.csv", # ref: https://developer.mozilla.org/es/docs/Web/HTTP/Headers/Content-Disposition
>           "Content-Type": "text/csv"
>       }
>   )
> ```

Thank you answer~
